components/Ui/UiJoinWithCards.vue

```diff
@@ -34,6 +35,7 @@ interface Card {
   name: string;
   description: string;
   icon: string;
+  detail: string;
```

Comment:
This will cause an error in the **advocates** page, where we also use this component.